content/release-10-17-0/announcements-10-17-0.md

```diff
@@ -156,6 +156,19 @@ Cumulocity IoT supports new standard OAuth2 access tokens with the "at+jwt" type
 Single-sign-on users managed by new authorization servers (that is, generating OAuth2 access tokens with type "at+jwt") can thus login successfully into Cumulocity IoT.
 To process this new token type, Cumulocity IoT uses the [Nimbus JOSE + JWT](https://connect2id.com/products/nimbus-jose-jwt).
 
+### Microservice infrastructure changes
+
+#### Planned
+
+##### Breaking change: Use of Linux cgroup v2
+In a release higher than 10.18, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in version 2024 and higher it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process. 
```

Comment:
```suggestion
From release 2024, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in a version higher than 10.18 it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process. 
```

Reply:
@mgrumann I actually meant it this way around :-). I hope that is ok for you. We mention 2024 later down once more, do let´s go with it for now.